### PR TITLE
Added utilities for Android12 storage access

### PR DESCRIPTION
**Pull request by adrenak**
Updated README for permission granting.

### CODE REVIEW

File: Assets/MXR.SDK/Runtime/Android/Utils/MXRAndroidUtils.Device.cs [hunk 1]
```diff
@@ -7,6 +7,13 @@ public static partial class MXRAndroidUtils {
         public static AndroidJavaClass AndroidOSBuild =>
             new AndroidJavaClass("android.os.Build");
 
+        public static int AndroidSDKInt {
```
Comment (Aleyum) [inline Assets/MXR.SDK/Runtime/Android/Utils/MXRAndroidUtils.Device.cs:10]: Could we perhaps change this to `GetAndroidSDKAsInt`?

**Review by Aleyum — COMMENTED** [review]
LGTM

**Comment by Aleyum** [comment]
One comment, otherwise LGTM

**Review by johnmanagexr — APPROVED** [review]
Really nice cleanups here, thanks Vatsal! Looks good to me.

**Comment by adrenak** [comment]
Merging this. Thanks @Aleyum for the `AndroidSDKInt` pointer, better with an `As`

**Pull request merged**